app/views/layouts/_flashes.html.haml

```diff
@@ -1,25 +1,3 @@
 - flash.each do |key, value|
   - unless %w{hidden ga}.include? key
-    - if key == "alert"
-      .error-summary{role: "group", 'aria-labelledby': "error-summary-heading", tabindex:'-1'}
-        %h1.heading-medium.error-summary-heading{id: 'error-summary-heading'}
-          = value
-
-    - elsif key == "notice"
-      .notice-summary{role: "group", 'aria-labelledby': "notice-summary-heading", tabindex:'-1'}
-        %h1.heading-medium.notice-summary-heading{id: "notice-summary-heading"}
-          = value
-
-    - elsif key == "success"
-      .govuk-notification-banner.govuk-notification-banner--success{ role: 'alert', 'aria-labelledby': 'govuk-notification-banner-title', 'data-module': 'govuk-notification-banner' }
-        .govuk-notification-banner__header
-          %h2#govuk-notification-banner-title.govuk-notification-banner__title
-            = key.capitalize
-
-        .govuk-notification-banner__content
-          %p.govuk-notification-banner__heading
-            = value
-
-    - else
-      .flash{class: "#{key}" }
-        %span= value
```

Comment:
Are there any feature tests around this feature? And what could be the impact of losing the `else`?